src/Maths/BinaryConvert.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @function BinaryConvert
+ * @description Convert the decimal to binary.
```

Comment:
It converts a decimal to binary-in-decimal... Not sure how to phrase this.

Reply:
> It converts a decimal to binary-in-decimal... Not sure how to phrase this.

I think the description is fine, if you give the function 4 it will result with 100, which is 4 in binary